parsing/depend.ml

```diff
@@ -138,7 +138,7 @@ and add_package_type bv (lid, l) =
 (* CR layouts: Remember to add this when jkinds can have module
```

Comment:
I think this time is now.

Reply:
Good catch! I've also added a test, which turned up a pre-existing ~bug that we weren't traversing the parameters of type declarations. Which I guess is probably unreachable before now since they can't contain with bounds.